tests/ui/parallel-rustc/cycle_crash-issue-135870.stderr

```diff
@@ -10,9 +10,8 @@ note: ...which requires const-evaluating + checking `FOO`...
 LL | const FOO: usize = FOO;
    |                    ^^^
    = note: ...which again requires simplifying constant for the type system `FOO`, completing the cycle
-   = note: cycle used when running analysis passes on this crate
    = note: see https://rustc-dev-guide.rust-lang.org/overview.html#queries and https://rustc-dev-guide.rust-lang.org/query.html for more information
 
 error: aborting due to 1 previous error
 
-For more information about this error, try `rustc --explain E0391`.
+For more information about this error, try `rustc --explain E0391`.
```

Comment:
Why is the file-ending newline removed?

Reply:
To be perfectly honest: not the faintest idea why my changes affect this test at all 😵